nautobot/dcim/filters/__init__.py

```diff
@@ -1667,6 +1676,12 @@ class PowerFeedFilterSet(
         to_field_name="name",
         label="Power panel (name or ID)",
     )
+    destination_panel = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
Similarly, please add the new filters to the `PowerFeedTestCase.generic_filter_tests`.